workspace_host/interface.js

```diff
@@ -26,17 +26,11 @@ const socketServer = require('../lib/socket-server'); // must load socket server
 const { logger } = require('@prairielearn/logger');
 const LocalLock = require('../lib/local-lock');
 
-const config = require('../lib/config');
+const { config, loadConfigAsync } = require('../lib/config-new');
```

Comment:
In a future PR, I plan on separating the bits of the workspace config schema from the primary config schema, just like we did with the graders.